ga4gh/cli.py

```diff
@@ -260,6 +260,9 @@ def addGlobalOptions(parser):
         "--config-file", "-f", type=str, default=None,
         help="The configuration file to use")
     parser.add_argument(
+        "--tls", "-t", type=bool, default=False,
```

Comment:
This doesn't work as you might expect. To get a proper boolean flag, remove ``type=bool`` and add ``action="store_true"`` like the example immediately below.